components/layout_2020/flow/construct.rs

```diff
@@ -531,54 +469,25 @@ where
         contents: Contents,
         box_slot: BoxSlot<'dom>,
     ) {
-        // We just found a block level element, all ongoing inline level boxes
-        // need to be split around it. We iterate on the fragmented inline
-        // level box stack to take their contents and set their first_fragment
-        // field to false, for the fragmented inline level boxes that will
+        // We just found a block level element, all ongoing inline level boxes need to be split
+        // around it. We iterate on the fragmented inline level box stack to take their contents and
+        // set their first_fragment field to false, for the fragmented inline level boxes that will
```

Comment:
maybe we can remove this now as we don't have `fragmented_inline_boxes` and neither do we iterate on the fragment inline I think